### DISCUSSION TITLE
FeaturerRequest: Orders on accessories

### DISCUSSION THREAD

**Discussion by aj-bi** [snipe/snipe-it#12606] [Ideas]
I am currently recording an accessory with the quantity and date of purchase. For example, 10 "Logitech Keyboard 200", purchased on 01.02.2023 for 300 EUR.

If I now buy another 10 "Logitech Keyboard 200" on 01.05.2023 for 200 EUR, do I then increase the quantity of the accessory, or do I enter a new accessory?

If I enter a new accessory, it becomes confusing, if I simply change the quantity, the warranty and the price are no longer correct.

How is this handled correctly?

I would like a FeatureRequest or an explanation of how best to handle this so that I can enter "orders" on an accessory. The feature request could be used to track the orders on the accessory.

What do you think?

**Comment by TreeBranches** [comment]
Giving this a prod as it feels like it's fallen by the wayside. We've been including Purchase Dates/Costs/Quantity in the Notes section which is... not great.

**Comment by aj-bi** [comment]
This thread goes in the same direction:

https://github.com/grokability/snipe-it/discussions/16605
